spec/features/borrower_creates_loan_request.rb

```diff
@@ -0,0 +1,33 @@
+require "rails_helper"
+
+RSpec.feature "borrower creates loan request" do
+  let(:borrower) { User.create(name: "Richard", email: "richard@example.com", password: "password", role: "borrower") }
+
+  def create_loan_request(title, description, amount, rate)
```

Comment:
Unused method argument - `rate`. If it's necessary, use `_` or `_rate` as an argument name to indicate that it won't be used.